_includes/sidebar.html

```diff
@@ -12,15 +12,22 @@
     <nav class="sidebar-nav">
       {% for sections in site.contents[page.lang] %}
         {% assign _section = sections[0] | strip %}
+        {% if page.section == _section %}
+          {% assign opened = true %}
+        {% elsif page.section == "home" and forloop.index0 == 1 %}
+          {% assign opened = true %}
```

Comment:
I would prefer to go with `_section == 'basics'` instead of `forloop.index0 == 1`
The 2 clauses could fit in one, as in:
```
        {% if page.section == _section or (page.section == "home" and _section == "basics") %}
￼           {% assign opened = true %}
```